BankSDK/GiniBankSDKPinning/Pod/README.md

```diff
@@ -4,17 +4,40 @@ Distributing Gini Bank SDK Pinning XCFrameworks using CocoaPods
 How to release new versions
 ---------------------------
 
-1. After the `.xcframeworks` have been created compress them into a zip named `GiniBankSDKPinning-XCFrameworks.zip`.
+1. After the `.xcframeworks` have been created compress them into a zip named `GiniBankSDKPinning-XCFrameworks.zip`.  
```

Comment:
[suggestion] We can change the steps starting with step 2 with:
2. Clone the https://github.com/gini/gini-podspecs repository.
3. Go into the `gini-podspecs` repository folder and create a subfolder in `GiniBankSDKPinning`which is named after the version, for example 2.2.1.
4. Copy the `GiniBankSDKPinning-XCFrameworks.zip` into the new folder.
5. Copy the `GiniBankSDKPinning.podspec` file from a previous version and update the version with the corresponding one, for example spec.version = 2.2.1.
6. Commit and push the new folder and its contents to `gini-podspecs` repo.